modules/ROOT/pages/fd-reuse-config.adoc

```diff
@@ -0,0 +1,65 @@
+= Create a Connection for Reuse by Yourself or Others
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can create, change and reuse a connection in a Flow Designer project instead of adding your authentication information manually. Once a reusable connection is created, it is available for all Flow Designer projects.
+
+You can reuse connections:
+
+* If you are a system admin, you can enable non-admins to connect to your system using administrative authentication. This is important if the integration requires administrative system permissions.
+
+* If you are responsible for maintaining the connection, you can ensure that the integration is unaffected when the user who created the Flow Designer project changes their personal password or leaves the organization.
+
+== Create a Reusable Connection
+
+Adding a reusable connection is available to you when configuring a connector in Flow Designer.
+
+. Once you are viewing the Flow Designer canvas, click *+* to add a card.  A list of the available connectors displays within the card.
+
+. In the Search field, type the name of the connector and select it. The connection section of the connector displays a list of operations that you can use to trigger Mule applications with the connector.
```

Comment:
Do you think this whole thing can be shortened to "Choose an operation."?

Reply:
Ha! Yeah, that's a win in the minimalist category.